src/frontends/tensorflow/src/op/if.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "input_model.hpp"
+#include "op_table.hpp"
+#include "openvino/opsets/opset10.hpp"
+
+using namespace std;
+using namespace ov;
+using namespace ov::opset10;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_if_op(const NodeContext& node) {
+    default_op_checks(node, 1, {"If", "StatelessIf"});
+    auto node_name = node.get_name();
+    auto translate_session = node.get_translate_session();
+    FRONT_END_GENERAL_CHECK(translate_session, "[TensorFlow Frontend] Internal error: Translate session is nullptr.");
+
+    // retrieve body ov::Model for then and else branches
+    auto then_branch_type = node.get_attribute<std::string>("then_branch");
```

Comment:
The body graph function name can be any. Of course, excluding empty string and with special characters (like `:`) but we don't perform this check because it mean that someone corrupts the model. `translate_session->get_body_ov_model(then_branch_type);` will return `nullptr` and TF FE complains with the error: `"[TensorFlow Frontend] Internal error or incorrect input model. Cannot find branch body graph with name "`